include/h2o/dsr.h

```diff
@@ -35,24 +35,35 @@
  */
 typedef struct st_h2o_dsr_req_t {
     /**
-     * The transport protocol for which DSR is performed (e.g., quic-29). This value indicates how the packet should be protected,
-     * aside from the cipher-suite being used.
+     * HTTP version of front-end connection
      */
-    uint32_t quic_version;
-    /**
-     * The cipher-suite being used, as the ID registered to the TLS Cipher Suites registry.
-     */
-    uint16_t cipher;
-    /**
-     * address from which QUIC packets should be sent
-     */
-    quicly_address_t address;
+    int http_version;
```

Comment:
```suggestion
    uint32_t http_version;
```
We are going to serialize this value, assuming that it takes no more than X characters. Therefore it is better to have the exact size of the value being specified.